astro/src/content/docs/get-started/use-cases/authorization-hub.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Authorization Hub
+description: Integrate with third party platforms, using FusionAuth to safely store refresh tokens.
+section: get started
+subcategory: use cases
+---
+import Aside from 'src/components/Aside.astro';
+import StoreRefreshTokensDiagram from 'src/diagrams/docs/get-started/use-cases/authorization-hub/store-refresh-token.astro';
+import RetrieveAccessTokenDiagram from 'src/diagrams/docs/get-started/use-cases/authorization-hub/retrieve-access-token.astro';
+import AuthorizationHubDescription from 'src/content/docs/get-started/use-cases/_authorization-hub-description.mdx';
+
+## Overview
+
+<AuthorizationHubDescription />
+
+## Problem 
+
+You have an application that leverages social or other platform providers, using their APIs to provide functionality to your users.
+
+You need to manage your users' tokens in one place, as well as easily add new integrations.
+
+With the authorization hub implementation, you can manage tokens for social providers and third party platforms such as:
+
+* Google
+* YouTube
+* Facebook
+* Instagram
+* LinkedIn
+* Microsoft Entra Id
+* Any provider supporting OIDC
+
+## Solution
+
+Use FusionAuth as your hub for this functionality. FusionAuth can be a centralized repository for long lived tokens and make it easier to integrate with third party platforms.
+
+You can use FusionAuth as:
+
+* an adapter for social provider integrations, making it easy to add new ones and stay up to date as the providers change
+* a storage location, safely holding long-lived tokens for each user, making them available to your application via a secure API or SDK
+
+This is an example of [the Third-Party Service Authorization Mode](/docs/lifecycle/authenticate-users/oauth/modes#third-party-service-authorization).
+
+## Prerequisites
+
+You have configured your application or applications to delegate authentication to FusionAuth via the Authorization Code grant and the hosted login pages.
+
+You can also implement this use case without using the hosted login pages, using the FusionAuth APIs. The API you'd use is the `Complete the Login` [Identity Provider APIs](/docs/apis/identity-providers/), but that implementation is beyond the scope of this document.
```

Comment:
Because that is the section of the identity provider APIs that would be used if you were building your own UX but wanted to use FusionAuth to manage the Identity Provider link.